04-make_epochs.py

```diff
@@ -67,9 +67,24 @@ def run_epochs(subject):
     if config.plot:
         epochs.plot()
 
-    print('  Writing to disk')
-    epochs.save(op.join(meg_subject_dir, '%s-epo.fif' % subject))
-
+    print('  Writing epochs to disk')
+    epochs_fname = op.join(meg_subject_dir,
+                            config.base_epochs_fname.format(**locals()))
+
+    epochs.save(epochs_fname)
+    
+    # produce high-pass filtered version of the data for ICA    
+    epochs_for_ICA = mne.Epochs(raw.copy().filter(l_freq=1., h_freq=None),
+                                events, config.event_id, config.tmin, 
+                                config.tmax, proj=True, 
+                                picks=picks, baseline=config.baseline,
+                                preload=False, decim=config.decim,
+                                reject=config.reject)
+    
+    
+    epochs_for_ICA_fname = op.splitext(epochs_fname)[0] + '_for_ICA.fif'
```

Comment:
I think the ICA code should filter the data in memory and the otuput of the ICA script should only be a -ica.fif file. No need to duplicate epochs etc. on disk.

Reply:
But since we are running the ICA on epoched data, there is no raw data to be filtered in memory. And the epochs are too short for 1Hz hp.